### PR TITLE
Fix float timestamp

### PR DESCRIPTION
**Pull request by bngreen**

### CODE REVIEW

File: src/claims.rs [hunk 1]
```diff
@@ -247,6 +247,15 @@ mod serde_timestamp {
                 .single()
                 .ok_or_else(|| E::custom("UTC timestamp overflow"))
         }
+
+        fn visit_f64<E>(self, value: f64) -> Result<Self::Value, E>
```
Comment (slowli) [inline src/claims.rs:251]: Please add `#[allow(clippy::cast_possible_truncation)]` on this method and an explanation why truncation is OK in this case (`Utc.timestamp_opt()` will return no value in this case anyway).

File: src/claims.rs [hunk 2]
```diff
@@ -353,4 +363,13 @@ mod tests {
             ValidationError::NotMature
         );
     }
+    #[test]
+    fn float_timestamp() {
+        let claims = "{\"exp\": 1.691203462e+9}";
+        let claims: Result<Claims<Empty>, _> = serde_json::from_str(claims);
+        assert!(claims.is_ok());
```
Comment (slowli) [inline src/claims.rs:370]: Nit: You can replace this with a more concise line:

```rust
let claims: Claims<Empty> = serde_json::from_str(claims).unwrap();
```

File: src/claims.rs [hunk 3]
```diff
@@ -353,4 +363,13 @@ mod tests {
             ValidationError::NotMature
         );
     }
+    #[test]
+    fn float_timestamp() {
```
Comment (slowli) [inline src/claims.rs:367]: You can also add a couple of overflowing timestamps (such as `{"exp": 1e64}` and `{"exp": -1e64}`) to test that they lead to a deserialization error.

**Review by slowli — COMMENTED** [review]
Terribly sorry with a delayed review, I've overlooked the GitHub notification somehow.

**Comment by IsraelZeromski** [comment]
Hi, is possible build a new version with this fix or when you plane to take out release a new one. Recently i found this problem

**Pull request merged**